tests/ui/conftest.py

```diff
@@ -14,31 +24,110 @@ def capabilities(capabilities):
     capabilities['marionette'] = True
     return capabilities
 
+
 @pytest.fixture
 def fxa_account(base_url):
-    url = DEV_URL if 'dev' in base_url else PROD_URL
+    url = DEV_URL if 'dev' or 'localhost' in base_url else PROD_URL
     return FxATestAccount(url)
 
 
-@pytest.fixture(scope='session')
-def jwt_issuer(base_url, variables):
+@pytest.fixture
+def jwt_issuer(base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_issuer']
+        hostname = urlparse.urlsplit(base_url).hostname
+        return json_file['api'][hostname]['jwt_issuer']
     except KeyError:
         return os.getenv('JWT_ISSUER')
 
 
-@pytest.fixture(scope='session')
-def jwt_secret(base_url, variables):
+@pytest.fixture
+def jwt_secret(base_url, json_file):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
-        return variables['api'][hostname]['jwt_secret']
+        hostname = urlparse.urlsplit(base_url).hostname
+        return json_file['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
+def initial_data(transactional_db, live_server):
+    call_command('generate_addons', 10, app='firefox')
+
+
+@pytest.fixture
+def create_superuser(transactional_db, live_server, base_url, tmpdir):
+    hostname = urlparse.urlsplit(base_url).hostname
+    create_switch('super-create-accounts')
+    call_command('loaddata', 'initial.json')
+    temp = tmpdir.join('variables.json')
+
+    call_command(
+        'createsuperuser',
+        interactive=False,
+        username='uitest',
+        email='uitester@mozilla.org',
+        add_to_supercreate_group=True,
+        save_api_credentials=str(temp),
+        hostname=hostname
+    )
+
+
+@pytest.fixture
+def force_user_login():
+    from olympia.users.models import UserProfile
+    user = UserProfile.objects.get(username='uitest')
+    return user
+
+
+@pytest.fixture
+def user(
+        transactional_db, create_superuser, live_server, base_url,
+        fxa_account, jwt_token):
```

Comment:
This should probably look like

```python
def user(transactional_db, create_superuser, live_server, base_url,
         fxa_accont, jwt_token):
```

to be in-sync with our general code-style.